app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -143,7 +144,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       tr.classList.add('mc-tr')
 
       var cellsOutput = ''
-      var allTheTds = allTheTbodyTrs[i].querySelectorAll('td')
+      var allTheTds = allTheTbodyTrs[i].querySelectorAll('th, td')
```

Comment:
Likewise, this is more of an allTheBodyCells now

Reply:
All the things! Fixed.